menu_drop.js

```diff
@@ -0,0 +1,35 @@
+const menu = document.querySelector('.bars');
+const slideIn = document.querySelector('.header-desktop');
+const close = document.querySelector('.fa-xmark');
+const bottomLine = document.querySelector('.bottom_line');
+const menuLink = document.querySelectorAll('.menu_link');
+
+// ==========OPEN MENU===============
+menu.addEventListener('click', () => {
+  slideIn.style.width = '100vw';
+  bottomLine.style.display = 'block';
+  close.style.display = 'block';
+  menuLink.forEach((element) => {
+    element.style.display = 'block';
+  });
+});
+
+// =============CLOSE MENU=============
+close.addEventListener('click', () => {
+  slideIn.style.width = '0';
+  bottomLine.style.display = 'none';
+  close.style.display = 'none';
+  menuLink.forEach((element) => {
+    element.style.display = 'none';
+  });
+});
+
+// ========BACKGROUND CLOSE==============
+slideIn.addEventListener('click', () => {
+  slideIn.style.width = '0';
+  bottomLine.style.display = 'none';
+  close.style.display = 'none';
+  menuLink.forEach((element) => {
+    element.style.display = 'none';
+  });
+});
```

Comment:
- [ ] For some reason when your project is on mobile view, I am not able to scroll up or down. Kindly look into why this is the case.

[Screencast from 06-02-2023 03:32:32 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/216972405-0008a7fb-300b-4892-a3bf-19e791508cd7.webm)